staging/src/k8s.io/apiserver/pkg/storage/etcd3/metrics/metrics.go

```diff
@@ -85,6 +86,46 @@ var (
 		},
 		[]string{},
 	)
+	listStorageCount = compbasemetrics.NewCounterVec(
+		&compbasemetrics.CounterOpts{
+			Name:           "apiserver_list_storage_count",
+			Help:           "Number of LIST requests served from storage, split by resource and predicate complexity",
+			StabilityLevel: compbasemetrics.ALPHA,
+		},
+		[]string{"resource", "predicate_complexity"},
+	)
+	listStorageQueries = compbasemetrics.NewCounterVec(
+		&compbasemetrics.CounterOpts{
+			Name:           "apiserver_list_storage_queries",
+			Help:           "Number of storage range queries used to satisfy a LIST request, split by resource",
+			StabilityLevel: compbasemetrics.ALPHA,
+		},
+		[]string{"resource"},
+	)
+	listStorageNumFetched = compbasemetrics.NewCounterVec(
+		&compbasemetrics.CounterOpts{
+			Name:           "apiserver_list_storage_num_fetched",
```

Comment:
this is a weird metric... why would we want the cumulative count of objects we've read from lists. Like if I have 200 objects for resource A, and I list three times, this should store 600, but what good is this?